### PR TITLE
Marathi Openslr female speaker ASR

### PR DESCRIPTION
**Pull request by shubhamphal**
Self supervised representations with Wav2Vec2 and Language Model for decoding

### CODE REVIEW

**Comment by sw005320** [comment]
@simpleoier, there are two PRs for the same database.
Both of them passed the CI check.
Just FYI.

@shubhamphal, I'll give you more concrete instructions on how to merge your PR later.

File: egs2/mr_openslr64/asr1/conf/decode.yaml [hunk 1]
```diff
@@ -0,0 +1,122 @@
+log_level: INFO
```
Comment (ftshijt) [inline egs2/mr_openslr64/asr1/conf/decode.yaml:1]: I don't think this is a correct formatted `decode.yaml` please double check

File: egs2/mr_openslr64/asr1/conf/lm_config.yaml [hunk 2]
```diff
@@ -0,0 +1,122 @@
+log_level: INFO
```
Comment (ftshijt) [inline egs2/mr_openslr64/asr1/conf/lm_config.yaml:1]: ditto

File: egs2/mr_openslr64/asr1/conf/tuning/train_asr_conformer5.yaml [hunk 3]
```diff
@@ -0,0 +1,109 @@
+cudnn_benchmark: false
```
Comment (ftshijt) [inline egs2/mr_openslr64/asr1/conf/tuning/train_asr_conformer5.yaml:1]: Can you simplify the config and combine some of them? We expect the config to be aligned with the results you report

File: egs2/mr_openslr64/asr1/setup.sh [hunk 4]
```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
```
Comment (ftshijt) [inline egs2/mr_openslr64/asr1/setup.sh:1]: no need for this file

**Review by ftshijt — COMMENTED** [review]
Please carefully review the file needed and remove those unrelated files

**Pull request closed without merging**